test/e2e/ig_latency_GetPlacesIn.csv

```diff
@@ -0,0 +1 @@
+GetPlacesIn,2.244155,3.795856
```

Comment:
Can we add header?

Reply:
This is intended to be a temp file. Later on, I plan to concatenate all the ~30 CSV files into a single CSV file. Header will be added at that time.

Reply:
Or, maybe we can just leave it as it is -- each API has its own latency CSV file? If so, yea could add a header.

Reply:
A wrapper script to collect all the CSVs in one place SG.  After import-group rollout, this can act as a general perf regression suite, where we can report % diffs against checked-in version ...